CoreErrorReportView/src/au/gov/asd/tac/constellation/views/errorreport/ErrorReportTopComponent.java

```diff
@@ -0,0 +1,894 @@
+                                    updateSettingsIcon(severePopupAllowed, "#ef3030", "#ff5858");                                        
+                                }
+                                if (errorPopupLevels.contains("WARNING")) {
+                                    updateSettingsIcon(warnPopupAllowed, "#e88830", "#ff9848");
+                                }
+                                if (errorPopupLevels.contains("INFO")) {
+                                    updateSettingsIcon(infoPopupAllowed, "#dada40", "#eaea58");
+                                }
+                                if (errorPopupLevels.contains("FINE")) {
+                                    updateSettingsIcon(finePopupAllowed, "#40d0d0", "#58e0e0");
+                                }
+                            });
+                            Thread.sleep(750);
+                            SwingUtilities.invokeAndWait(() -> {
+                                setIcon(defaultIcon);
+                                if (errorReportLevels.contains("SEVERE")) {
+                                    updateSettingsIcon(severeReportFilter, "black", "black");                                     
+                                }
+                                if (errorReportLevels.contains("WARNING")) {
+                                    updateSettingsIcon(warnReportFilter, "black", "black");
+                                }
+                                if (errorReportLevels.contains("INFO")) {
+                                    updateSettingsIcon(infoReportFilter, "black", "black");
+                                }
+                                if (errorReportLevels.contains("FINE")) {
+                                    updateSettingsIcon(fineReportFilter, "black", "black");
+                                }
+                                
+                                if (errorPopupLevels.contains("SEVERE")) {
+                                    updateSettingsIcon(severePopupAllowed, "black", "black");
+                                }
+                                if (errorPopupLevels.contains("WARNING")) {
+                                    updateSettingsIcon(warnPopupAllowed, "black", "black");
+                                }
+                                if (errorPopupLevels.contains("INFO")) {
+                                    updateSettingsIcon(infoPopupAllowed, "black", "black");
+                                }
+                                if (errorPopupLevels.contains("FINE")) {
+                                    updateSettingsIcon(finePopupAllowed, "black", "black");
+                                }
+                            });
+                            Thread.sleep(650);
+                        } catch (final InvocationTargetException | InterruptedException ex) {
+                            Exceptions.printStackTrace(ex);
```

Comment:
use a log entry instead of a stack trace print (for interrupt exception also call the thread interrupt)